README.md

```diff
@@ -146,18 +156,32 @@ $ helm dependency update
 
 After doing this, helm will download the `spring-boot-example-app` chart from the repository `http://snowdrop.github.io/helm` and will copy the Chart tallball `spring-boot-example-app-0.0.3.tgz` at `my-custom-chart/charts/`.
 
-As we explained in [the usage section](#spring-boot-example-app) of the `spring-boot-example-app` chart, we need to provide the GIT repository and branch, so Openshift can build and deploy the Spring Boot application using S2i. For doing this, we need to edit the `my-custom-chart/values.yaml` file and provide this configuration:
+As we explained in [the usage section](#spring-boot-example-app) of the `spring-boot-example-app` chart, for Openshift we need to provide the GIT repository and branch, so Openshift can build and deploy the Spring Boot application using S2i. For doing this, we need to edit the `my-custom-chart/values.yaml` file and provide this configuration:
```

Comment:
Text will be more readable if you say something like this ==> "As the build is taking place on the cluster using OpenShift - see usage section, then we have to configure the `s2i` fields defined within the `values file ..."

Reply:
changed